docs/reference/index.md

```diff
@@ -0,0 +1,16 @@
+<!--[metadata]>
++++
+title = "Compose CLI reference"
+description = "Compose CLI reference"
+keywords = ["fig, composition, compose, docker, orchestration, cli,  reference"]
+[menu.main]
+identifier = "smn_compose_cli"
+parent = "smn_compose_ref"	
++++
+<![end-metadata]-->
+
+## Compose CLI reference
+
+The following pages describe the usage information for the Docker Compose subcommands. You can also see this information by running `docker-compose [SUBCOMMAND] --help` from the command line.
```

Comment:
When we say "The following pages", where are these pages going to be listed? Is it just in the sidebar?